pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${bdg-formats.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.bdgenomics.qc-metrics</groupId>
+      <artifactId>qc-metrics-core</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
IntelliJ is not understanding where this comes from for me, even though cmdline-Maven finds it

![](https://s3.amazonaws.com/f.cl.ly/items/0l2d2J2t1P0T2t431b2c/Screen%20Shot%202015-07-01%20at%202.57.11%20PM.png)

is IJ working for you here?